backend/loadbalancer.go

```diff
@@ -14,15 +15,16 @@ const minRequiredBackends = 2
 type LoadBalancerNode struct {
 	backend wasabi.RequestHandler
 	counter atomic.Int32
+	weight  uint
 }
 
 type LoadBalancer struct {
 	backends []*LoadBalancerNode
 }
 
 // NewLoadBalancer creates a new instance of LoadBalancer with the given backends.
-// It takes a slice of RequestHandler as a parameter and returns a new instance of LoadBalancer.
-func NewLoadBalancer(backends []wasabi.RequestHandler) (*LoadBalancer, error) {
+// It takes a slice of `WeightedBackend` as a parameter and returns a new instance of LoadBalancer.
```

Comment:
Could you mentioned here that backends with `0` weight will be ignored.

Reply:
ok